storage/src/cache/mod.rs

```diff
@@ -36,6 +36,8 @@ use crate::utils::{alloc_buf, check_digest};
 use crate::{StorageResult, RAFS_MAX_CHUNK_SIZE};
 
 mod cachedfile;
+#[cfg(feature = "dedup")]
```

Comment:
Please include tests of this mod in ci.

Reply:
ci will be covered in next PR, which enables the chunk deduplication.

Reply:
> ci will be covered in next PR, which enables the chunk deduplication.

That is unacceptable. If this pr is merged with next pr referred, then acceptable.

Reply:
turn on unit test for dedup by enabling `dedup` feature by default.

Reply:
LGTM.